automation/repeated_test.sh

```diff
@@ -123,7 +129,21 @@ if (( $# > 0 )); then
         shift
     done
 else
-    mapfile -t TEST_LANES < <( find cluster-up/cluster -name 'k8s-[0-9].[0-9][0-9]' -print | sort -rV | grep -oE 'k8s.*' | head -1 )
+    # We only want to use stable providers for flake testing, thus we fetch the k8s version file from kubevirtci.
+    # we stop at the first provider that is stable (aka doesn't have an rc or beta or alpha version)
+    for k8s_provider in $(find cluster-up/cluster -name 'k8s-[0-9].[0-9][0-9]' -print | sort -rV | grep -oE 'k8s.*'); do
```

Comment:
```suggestion
    for k8s_provider in $(cd cluster-up/cluster && ls -rd k8s-[0-9]\.[0-9][0-9]); do
```

Reply:
TIL: `ls -rd` sorts the directories in reverse order like `sort -rV`.

Thanks!

Reply:
With the caveat that `ls` does an alphabetical sort, not a version sort. So if/when we get to k8s version `1.100` or `2.10`, they'll be considered older than resp. `1.99` and `2.9`.
If you want to be that future-proof, you might want to still do `ls -d | sort -rV`.